Tests/SonarQube.Bootstrapper.Tests/BootstrapperExeTests.cs

```diff
@@ -161,6 +162,44 @@ public void Exe_PreProc_VersionCheckSucceeds_PreProcFails()
         }
 
         [TestMethod]
+        public void Exe_PreProc_InstallTargets()
```

Comment:
Hmm, I don't think we ensure that the pre/post processor do not install the targets. Do you see it as being a risk?